README.md

```diff
@@ -17,7 +17,7 @@ The protocol allows a "client" and "server" to establish a bidirectional connect
 Requires Node >= v8.10.
 
 ```sh
-npm install --save ilp-protocol-stream
+npm install ilp-protocol-stream ilp-plugin
```

Comment:
Since that's only needed for the examples I'd rather have that listed separately. Explicitly call out (either in the installation section or the example section) that you also need that if you want to run the examples